python/samples/kernel-syntax-examples/self-critique_rag.py

```diff
@@ -48,7 +48,9 @@ async def main() -> None:
     kernel.add_text_completion_service(
         "dv",
         AzureTextCompletion(
-            deployment_name="gpt-35-turbo-instruct",
```

Comment:
why was this undone?

Reply:
in the subscription that is used for integration tests, we don't have a `gpt-35-turbo-instruct` model defined. I was testing it, and can revert.